rdbi-recipes/src/main/java/com/lithium/dbi/rdbi/recipes/scheduler/MultiChannelScheduler.java

```diff
@@ -116,6 +118,7 @@ public boolean schedule(String channel, String tube, final String job, final int
                     getMultiChannelCircularBuffer(tube),
                     getMultiChannelSet(tube),
                     getRunningQueue(tube),
+                    getPerChannelTrackingEnabled(),
```

Comment:
can't we just pass the isPerChannelTrackingEnabled() instead of the key?

Reply:
we could but it could create a few issues -- for one we have to make a separate call to check it, so that then reserving would require 2x calls (as opposed to the script which gets totally executed server side). it also exposes us to a potential race condition, so we check enabled, then another process toggles it, then we send in a stale value here.